src/app/components/moj/moj-pagination/moj-pagination-list/moj-pagination-list.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-moj-pagination-list',
```

Comment:
Change this: selector: 'app-moj-pagination-list',
To this: selector: 'app-moj-pagination-list, [app-moj-pagination-list]',